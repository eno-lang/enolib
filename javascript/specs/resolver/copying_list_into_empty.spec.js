const { parse } = require('../..');

const input = `
languages:
- eno
- json

copy < languages
`.trim();

describe('Resolution', () => {
  describe('Copying list into empty', () => {
    it('works', () => {
      const doc = parse(input);
      expect(doc.raw()).toMatchSnapshot();
    });
  });
});
