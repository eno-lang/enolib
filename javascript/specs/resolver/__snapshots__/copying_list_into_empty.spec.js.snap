// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resolution Copying list into empty works 1`] = `
Object {
  "elements": Array [
    Object {
      "items": Array [
        Object {
          "type": "listItem",
          "value": "eno",
        },
        Object {
          "type": "listItem",
          "value": "json",
        },
      ],
      "key": "languages",
      "type": "list",
    },
    Object {
      "items": Array [
        Object {
          "type": "listItem",
          "value": "eno",
        },
        Object {
          "type": "listItem",
          "value": "json",
        },
      ],
      "key": "copy",
      "type": "list",
    },
  ],
  "type": "document",
}
`;
